.github/workflows/benchmark.yml

```
@@ -0,0 +1,33 @@
+name: Benchmark
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+env:
+  DEFAULT_GO_VERSION: 1.17
```

It looks like this went back to Go 1.17.

```suggestion
  DEFAULT_GO_VERSION: 1.16
```